services/Cargo.toml

```diff
@@ -47,6 +47,7 @@ image = "0.24"
 lazy_static = "1.4"
 log = "0.4"
 mime = "0.3"
+nix = "0.25"
```

Comment:
Doesn't tie this the program to linux only?

Reply:
yes, windows sockets work differently. We have to think about whether we want a different solution for windows, support workflow cancellation only for linux, or tie geo engine to linux entirely. I haven't been able to get geo engine working on windows and I really tried. So we wouldn't really lose anything here.

Reply:
I made the dependency contingent on the target platform being linux